File: Лаба2/Задание 3 вариант 3/main.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("", (input) => {
    const inputs = input.split(' ');
    const n = parseInt(inputs[0]);
    let count = 0;

    for (let i = 1; i <= n; i++) {
        const g = parseInt(inputs[i]);
        let number = g;
        while (number > 0) {
            const h = number % 10;
            number = Math.floor(number / 10);
            if (h % 3 === 0) {
                count += h;
            }
        }
    }

    console.log(count);
    rl.close();
});

File: Лаба2/Задание 3 вариант 3/main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] inputs = scanner.nextLine().split(" ");
        int n = Integer.parseInt(inputs[0]);
        
        int count = 0;
        for (int i = 1; i <= n; i++) {
            int g = Integer.parseInt(inputs[i]);
            int number = g;
            while (number > 0) {
                int h = number % 10;
                number /= 10;
                if (h % 3 == 0) {
                    count += h;
                }
            }
        }
        
        System.out.println(count);
        scanner.close();
    }
}
